fs/rpmsgfs/rpmsgfs.c

```diff
@@ -252,6 +259,21 @@ static void rpmsgfs_mkpath(FAR struct rpmsgfs_mountpt_s  *fs,
     {
       strncat(path, &relpath[first], pathlen - strlen(path) - 1);
     }
+
+  while (fs->timeout > 0)
```

Comment:
Does `INT_MAX` timeout means "wait forever"?

Reply:
Almost, 0x7fffffff/1000/60/60/24=24.5days.

Reply:
Haha... The question is do we support infinite wait somehow or any value is considered a timeout?

Reply:
The current code handles any positive value as a timeout, so no support of infinite wait. I just want to make sure that it is by design

Reply:
Yes.
That I found the default value setting in wrong place, so update it.